### PR TITLE
Run storage_formats product test group in main run

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -64,12 +64,12 @@ script:
   - |
     if [[ -v PRODUCT_TESTS ]]; then
       presto-product-tests/bin/run_on_docker.sh \
-        multinode -x quarantine,big_query,storage_formats,profile_specific_tests,tpcds
+        multinode -x quarantine,profile_specific_tests,tpcds
     fi
   - |
     if [[ -v PRODUCT_TESTS ]]; then
       presto-product-tests/bin/run_on_docker.sh \
-        singlenode-kerberos-hdfs-impersonation -g storage_formats,cli,hdfs_impersonation,authorization
```
Comment (ArturGajowy) [inline .travis.yml]: I think the storage_formats tests were somehow important for testing kerberos - at last that's what I think I've heard. @cawallin @sanjay990 please chime in

**Comment by electrum** [comment]
They seem to only test storage formats, nothing to do with Kerberos or authorization / grants. The motivation here is simplification: if they work fine for `multinode` and don't increase runtime, it makes the configs simpler.

File: .travis.yml (hunk 1 above)
Comment (cawallin) [inline .travis.yml]: Hm, I don't recall that. This change should be fine.

**Comment by electrum** [comment]
Rebase and kick the tests again

**Pull request closed without merging**